### PR TITLE
Changes in the guides for some misspelling words

### PR DESCRIPTION
**Pull request by adrnliaw**
✅ I have read the [contributing file](https://github.com/NMF-earth/nmf-app/blob/main/contributing.md)

## Summary

I found some misspelling words for English language in the guides.json. Thus, I corrected those misspelling words.

## Changelog

nmf-app\assets\guides\guides.json

The main changes are in this file in which I corrected all the misspelling that I found.

## Demo

Firstly, I cloned the project into my disk and install dependencies using yarn and copied two files that contain secrets and replace them. Then, I tried to build the project with "yarn start" command. However, it failed due to dependencies issue. Therefore, I proceeded to installing necessary dependencies and build the project again, in which it turns out to be successful.

After that, I skim through the guides in the application and found some misspelling. Thus, I proceeded to correcting them.

After that, I tried to test the project. My first attempt in testing the project failed. Thus, I changed the "TZ=GMT jest" command in the test section of package.json file into "SET TZ=GMT && jest –runInBand -- forceExit" and try my second attempt in testing the project. The result was 302 out of 304 successful test and 2 snapshots failed. Then, I used the "yarn test -u" command to update the snapshot and get to pass all the tests. Then, I committed the project and push it to GitHub.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "web": "expo start --web",
     "eject": "expo eject",
     "test:watch": "jest --watchAll",
-    "test": "TZ=GMT jest",
+    "test": "SET TZ=GMT && jest --runInBand --forceExit",
```
Comment (PierreBresson) [inline package.json:11]: What are you trying to do with these changes? With these changes, `yarn test` no longer works. If you want to update the test (unfortunately it's needed at the moment) you need to run `yarn test -u`

File: package.json [hunk 2]
```diff
@@ -39,13 +39,13 @@
     "expo": "40.0.0",
     "expo-asset": "8.2.1",
     "expo-constants": "9.3.3",
-    "expo-document-picker": "9.1.2",
-    "expo-file-system": "11.0.2",
+    "expo-document-picker": "~8.4.1",
```
Comment (PierreBresson) [inline package.json:42]: Please no carret and no need to change version of the version of any package for wording fix

**Review by PierreBresson — CHANGES_REQUESTED** [review]
Thanks for the misspelling check. Need to revert some changes and we are good to go 👍

File: package.json (hunk 2 above)
Comment (adrnliaw) [reply package.json:42]: Okay, I will fix this. Thanks for informing.

File: package.json (hunk 1 above)
Comment (adrnliaw) [reply package.json:11]: Okay, I will change it back to how it was. Thanks for informing.

**Comment by PierreBresson** [comment]
@adrnliaw Thank for the pull request! 🙂

**Pull request merged**